src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -88,6 +89,14 @@ const FLIGHT_RULES = {
     IFR: 'ifr'
 };
 
+const AUTOTOWER_FLIGHT_PHASES = [
```

Comment:
Could you add a docblock here. specifically, why do we need this and how is it used? (I know these answers, but others coming through here may not.

Reply:
done